portfolio/src/main/java/com/google/sps/servlets/PostCommentServlet.java

```diff
@@ -39,12 +41,25 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment-input").trim();
     if (!text.isBlank()) { 
-      long commentPostTime = System.currentTimeMillis();
       Entity commentEntity = new Entity(Comment.ENTITY_NAME);
       commentEntity.setProperty(Comment.TEXT_FIELD, text);
+      long commentPostTime = System.currentTimeMillis();
       commentEntity.setProperty(Comment.POST_TIME_FIELD, commentPostTime);
+      String commentUserEmail = getUserEmail();
+      commentEntity.setProperty(Comment.USER_EMAIL_FIELD, commentUserEmail);
```

Comment:
```suggestion
      commentEntity.setProperty(Comment.USER_EMAIL_FIELD, getUserEmail());
```
nit: could be reduced to one line without sacrificing much readability

Reply:
yeah, that makes alot of sense